students/MikeShand/lesson07/test_html_render.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+
+import html_render as hr
+
+
+def test_element():
+    test1 = hr.Element('intro', color = 'blue')
+    test1.append('test text')
+    assert 'test text' and 'intro' in test1.content

```

Comment:
are you attempting to assert that both 'test text' & 'intro' are in the test1.content? if that is so then the above code does not do that

let me know, I may be misunderstanding